pkg/xray/util.go

```diff
@@ -160,27 +64,6 @@ func mkNames(name, resource string) (int, string, string) {
 	return id, fmt.Sprintf("%s.%s", resource, n), n
 }
 
-type Lens func(key string, value interface{}) []error
-
-func mkLens(d *schema.ResourceData) Lens {
-	var errors []error
-	return func(key string, value interface{}) []error {
-		if err := d.Set(key, value); err != nil {
-			errors = append(errors, err)
-		}
-		return errors
-	}
-}
-
-func sendConfigurationPatch(content []byte, m interface{}) error {
-
-	_, err := m.(*resty.Client).R().SetBody(content).
-		SetHeader("Content-Type", "application/yaml").
-		Patch("artifactory/api/system/configuration")
-
-	return err
-}
-
 var neverRetry = func(response *resty.Response, err error) bool {
```

Comment:
There's a `NeverRetry` func in the shared module. Let's re-use that.

Reply:
done